src/aioswitcher/bridge.py

```diff
@@ -32,6 +32,7 @@
     ShutterDirection,
     SwitcherBase,
     SwitcherPowerPlug,
+    SwitcherS12,
```

Comment:
This brakes our current design, we named devices by their function (e.g. Water Heater, Shutter, etc...)
This is a Shutter device (or maybe a dual shutter) but now we name it by the device model